proto/sentry_protos/snuba/v1/endpoint_time_series.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_attribute.proto";
+import "sentry_protos/snuba/v1/trace_item_filter.proto";
+
+enum Function {
+  FUNCTION_UNSPECIFIED = 0;
+  FUNCTION_SUM = 1;
+  FUNCTION_AVERAGE = 2;
+  FUNCTION_COUNT = 3;
+  FUNCTION_P50 = 4;
+  FUNCTION_P90 = 5;
+  FUNCTION_P95 = 6;
+  FUNCTION_P99 = 7;
+  FUNCTION_AVG = 8;
+  FUNCTION_MAX = 9;
+  FUNCTION_MIN = 10;
+  FUNCTION_UNIQ = 11;
+}
+
+message TimeSeriesRequest {
+  RequestMeta meta = 1;
+  TraceItemFilter filter = 2;
+  repeated AttributeAggregation aggregations = 3;
+  uint64 granularity_secs = 4;
+  repeated AttributeKey group_by = 5;
```

Comment:
what happens if we group by something with high cardinality? sentry itself has this "top n" functionality that makes it so it only returns at most like 10 groups with the remainder put in "other", is that going to be the behavior here?

Reply:
cc @wmak

Reply:
> Why would the query be ordered by anything other than time?
cc @evanh 

I dunno where the comment went, but this is why I'm asking for `order by` and `limit by` so we can use it with `group by`

Reply:
👍 that's fine with me, so to match the current behaviour today then we'll keep doing what we've already been doing which is;

1. make a `TraceItemTableRequest` to get the totals
2. Pass those totals as part of the filter here to `TimeSeriesRequest`
3. Make a parallel query with the inverse filter for the `other`

In which case just the group-by should be fine